pkg/cmd/propertiesNamespace_test.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPropertiesNamespaceCommandInCommandCollectionHasName(t *testing.T) {
```

Comment:
name of this test is suspect - it's checking far more than just the name.

Reply:
deleted "HasName" from test name